src/Components/Locations.js

```diff
@@ -0,0 +1,54 @@
+import axios from "axios";
+import React from "react";
+
+class Locations extends React.Component {
```

Comment:
Would love to also see you import {Component} and then you can just extend Component